simtools/schemas/integration_tests_config.metaschema.yml

```diff
@@ -0,0 +1,74 @@
+---
+$schema: http://json-schema.org/draft-06/schema#
+$ref: '#/definitions/SimtoolsIntegrationTestConfiguration'
+title: SimPipe integration test configuration metaschema
+description: YAML representation of integration test configuration metaschema
+version: 0.1.0
+name: integration_tests_config.metaschema
+type: object
+additionalProperties: false
+
+
+definitions:
+  SimtoolsIntegrationTestConfiguration:
+    type: object
+    additionalProperties: false
+    properties:
+      CTA_SIMPIPE:
+        "$ref": "#/definitions/CTASIMPIPE"
+    required:
+      - CTA_SIMPIPE
+    title: ""
+  CTASIMPIPE:
+    type: object
+    additionalProperties: false
+    properties:
+      APPLICATION:
+        type: string
+        description: "Name of simtools application"
+      TEST_NAME:
+        type: string
+        description: "Test name. Should be unique for this application."
+      MODEL_VERSION_USE_CURRENT:
+        type: boolean
+        description: |
+          "Skip test if model version is changed via command line.
+          Important e.g., for cases where DB model is incomplete for some
+          model versions."
+      CONFIGURATION:
+        "$ref": "#/definitions/Configuration"
+      INTEGRATION_TESTS:
+        type: array
+        items:
+          "$ref": "#/definitions/IntegrationTest"
+    required:
+      - APPLICATION
+      - CONFIGURATION
+      - TEST_NAME
+    title: CTASimpipe
+  Configuration:
+    description: "Command line configuration of simtools application."
+    type: object
+    additionalProperties: true
+    title: Configuration
+  IntegrationTest:
+    description: "List of integration tests."
+    type: object
+    additionalProperties: false
+    properties:
+      OUTPUT_FILE:
+        description: |
+          "Path of output file tested to be generated by integration test."
+        type: string
+      FILE_TYPE:
+        description: |
+          "Expected file type of output file generated by integration test."
```

Comment:
I think we actually don't need this anymore, do we? If we check whether the file exists (with the type in the file name), we will not need to test its type again.

Reply:
Yes - although I would keep it as a weaker version of the exact file name test ('whatever the output file name is, the type should be e.g., json').